src/main/java/com/jcraft/jsch/JSch.java

```diff
@@ -27,8 +27,12 @@
 package com.jcraft.jsch;
 
 import java.io.InputStream;
+import java.util.Collections;
 import java.util.Enumeration;
+import java.util.HashMap;
 import java.util.Hashtable;
+import java.util.Map;
+import java.util.Set;
```

Comment:
Can you remove this unused `java.util.Set` import that SonarQube is pointing out?